www/havm10/ukol2.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="cs">
+
+<head>
+    <meta charset="utf-8" />
+    <title>VŠE-pro-VŠE</title>
+</head>
+
+<body>
+    <header>
+        <span>VŠE-pro-VŠE</span>
+        <nav>
+            <a href="">Domovská stránka</a>
+            <br>
```

Comment:
Odstranil bych ty br, jelikoz to bude horizontalni menu

Reply:
opraveno